server/master/src/main/java/org/apache/accumulo/master/tableOps/create/CreateTable.java

```diff
@@ -62,18 +65,19 @@ public long isReady(long tid, Master environment) throws Exception {
 
     // assuming only the master process is creating tables
 
-    Utils.idLock.lock();
+    Utils.getIdLock().lock();
     try {
-      tableInfo.tableId = Utils.getNextId(tableInfo.tableName, master.getContext(), Table.ID::of);
+      tableInfo
+          .setTableId(Utils.getNextId(tableInfo.getTableName(), master.getContext(), Table.ID::of));
```

Comment:
This was probably autoformat'ed but you could put a line break somewhere else.

Reply:
Is there a way to turn off the auto formatter for that line cause I believe it's just gonna get reverted back the next time it runs.

Reply:
It's OK the the way it is, I'm not sure messing with the formatter looks much better anyway.  But to turn the formatter off and then back on requires these lines:
// @formatter:off
...
// @formatter:on

Reply:
I don't think it's worth messing with the formatter enabling/disabling here. Abuse of that can lead to some really ugly code over time. It looks fine to me as is.

Reply:
Also, I'd have to double check, but I think master *is* a context, so I'm not sure you need to pass in `master.getContext()` when you can just pass in `master`. If I'm right, that could shorten the line enough to make this look prettier.

Reply:
If that doesn't work you could shorten "tableInfo" to tInfo or just info

Reply:
Or create a local variable above:
<code>String name = tableInfo.getTableName();</code>
Which will save you a bunch of characters on that line.

Reply:
`master` wasn't a context, unfortunately. I just used a local variable to shorten the line so it won't auto format later down the road.